src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+const getCapitalizeName = (str) => {
+  let attrName = str[0].toUpperCase();
+
+  for (let i = 1; i < str.length; i++) {
+    if (str[i].toUpperCase() === str[i]) {
+      attrName += ` ${str[i]}`;
+    } else {
+      attrName += str[i];
+    }
+  }
+
+  return attrName;
+};
+
+[...inputs].forEach(input => {
```

Comment:
```suggestion
inputs.forEach(input => {
```
As your colleague has mentioned, collections have a method `forEach`